phpcs.xml

```diff
@@ -41,6 +41,9 @@
     <exclude-pattern>\.components-civictheme/*</exclude-pattern>
     <exclude-pattern>components_combined/*</exclude-pattern>
 
+    <!-- Exclude settings.php file. -->
+    <exclude-pattern>web\/sites\/default\/settings\.php</exclude-pattern>
```

Comment:
Some govcms lint issue on their settings file - didn't make sense from the repository perspective.